R/zchunk_L261.carbon_storage_USA.R

```diff
@@ -48,131 +52,193 @@ module_gcam.usa_L261.carbon_storage_USA_DISABLED <- function(command, ...) {
     L261.StubTech_C <- get_data(all_data, "L261.StubTech_C")
     L261.GlobalTechCoef_C <- get_data(all_data, "L261.GlobalTechCoef_C")
 
-    # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses vecpaste
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses repeat_and_add_vector
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # ===================================================
+    # Create a vector of FERC grid regions with non-zero storage curves
```

Comment:
It's not entirely clear to me why we're doing this and the next step. Could we add some detail in the comment?

Thanks.